### ISSUE TITLE
[a11y] Base table selector not visible on Safari

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1191]
**Describe the bug**
Safari is not showing the tables when mapping in the WB

**To Reproduce**
1. Try to map a base table to a data set in the workbench
2. See error

**Expected behavior**
I should be able to select a base table

**Screenshots**
<img width="359" alt="image" src="https://user-images.githubusercontent.com/37256050/158612270-67ff649c-df53-41b3-a0d7-133155cb22e6.png">

**Desktop (please complete the following information):**
 - macOS 12.3, Safari 15.4 (latest as of this report)

**Comment by maxxxxxdlp** [comment]
The actual problem is that the dialog opens up too short to fit the base table selector.
In most browsers the base table selector asserts its dominance and dictates the dailog size. In Safari ... that doesn't work.
I fixed this case in ae1d3a57 but there might be other cases like this.

@grantfitzsimmons When doing pre-release testing in Safari, please be on the lookout for more cases like this.
